### PR TITLE
Prettier and more colorized output for the apply commands

### PR DESCRIPTION
**Pull request by jilleJr**
I was fiddling around with the logging output. I'm used to zerolog, and know this little hack you can do by setting the `caller` field to get the text at the beginning (meant for source code line:number)

Then I started playing around with the coloring of the changes it can make, to make them stand out a bit more.

### Before

image

### After

image

### CODE REVIEW

**Comment by stefanprodan** [comment]
@jilleJr this looks wonderful, thank you!

Could we highlight better in the --dry-run the created/modified/deleted resources vs the unchanged ones, maybe we gray out the once with no changes?

**Comment by jilleJr** [comment]
> @jilleJr this looks wonderful, thank you!
> 
> Could we highlight better in the --dry-run the created/modified/deleted resources vs the unchanged ones, maybe we can gray out the once with no changes?

You mean like this?

Screenshot_20230711_161744

Or like this?

Screenshot_20230711_162143

File: cmd/timoni/log.go [hunk 1]
```diff
@@ -56,6 +62,118 @@ func NewConsoleLogger(pretty bool) logr.Logger {
...
+	colorCallerPrefix = color.New(color.FgHiBlack)
+	colorBundle       = color.New(color.FgHiMagenta)
+	colorInstance     = color.New(color.FgHiMagenta)
+	colorPerAction    = map[ssa.Action]*color.Color{
+		ssa.CreatedAction:    color.New(color.FgHiGreen),
+		ssa.ConfiguredAction: color.New(color.FgHiCyan),
+		ssa.UnchangedAction:  color.New(color.FgHiBlack),
+		ssa.DeletedAction:    color.New(color.FgRed),
+		ssa.SkippedAction:    color.New(color.FgHiBlack),
+		ssa.UnknownAction:    color.New(color.FgYellow),
```
Comment (stefanprodan) [inline cmd/timoni/log.go:76]: These look great, thank you!

**Comment by stefanprodan** [comment]
As a followup, if you have time, the same colouring could be used in `timoni status` and `timoni inspect resources`.

**Comment by jilleJr** [comment]
Sure! Sounds like some cozy coding

**Pull request merged**